.github/workflows/ci-additional.yml

```diff
@@ -74,6 +74,28 @@ jobs:
         shell: bash -l {0}
         run: pytest -v --doctest-modules --ignore-glob='*/test_*.py' dask
 
+  doctest-pip:
```

Comment:
I'm not sure if we care about this, but since it came up in https://github.com/dask/dask/issues/7358 I thought I'd try it out. I added a skip in conftest.py to support this.